make-it/EmMakefile-AAL

```diff
@@ -35,7 +35,7 @@ ifeq (,$(wildcard $(ANALYSIS_WRAPPERS_SOURCE_DIRECTORY)/AnalysisWrapper*.cpp))
 ANALYSIS_LIBRARY = $(OBJS)/NoAnalysisLibrary.bc
 $(OBJS)/NoAnalysisLibrary.bc: | no-analysis
 	touch $(OBJS)/NoAnalysis.cpp
-	$(EMCC) $(EM_ANALYSIS_FLAGS) -o $@ $(OBJS)/NoAnalysis.cpp
+	$(EMCC) $(EM_ANALYSIS_FLAGS) -o $@ $(OBJS)/NoAnalysis.cpp -c
```

Comment:
The `-c` flag runs a [subset of steps](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-c) (only run preprocess, compile, and assemble steps). Why was this change made?

Reply:
For the intermediate level, we need only object file, previously by default object files were generating. Now we have to explicitly mention the -c flag.

2.0.3 realese note snipet:
![image](https://user-images.githubusercontent.com/41286723/121939571-c7324d00-cd12-11eb-8e6d-9a6979b42f66.png)